src/components/Select/Select.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useMemo,useRef, useState } from 'react';
+
+import { Icon } from '../Icon';
+
+import {
+  SelectButtonWraper,
+  SelectItem,
+  SelectLabel,
+  SelectList,
+  SelectWraper,
+} from './styled';
+import { SelectInterface } from '.';
+
+export const Select = ({
+  label,
+  placeholder,
```

Comment:
```suggestion
  placeholder: 'Choose from the list',
```

Reply:
Accepted and fix, but i used it like that (not ":" only "=") like default placeholder
placeholder = 'Choose from the list',